perfkitbenchmarker/providers/profitbricks/flags.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from perfkitbenchmarker import flags
+
+
+# Locations
+US_LAS = 'us/las'
+DE_FKB = 'de/fkb'
+DE_FRA = 'de/fra'
+
+# Zones
+ZONE_1 = 'ZONE_1'
+ZONE_2 = 'ZONE_2'
+
+flags.DEFINE_string('profitbricks_config',
+                    os.getenv('PROFITBRICKS_CONFIG',
+                              '~/.config/profitbricks-auth.cfg'),
+                    ('Path to config file containing your email and password. '
+                     'Can also be set via $PROFITBRICKS_CONFIG environment '
+                     "variable.\n(File format: email:password)"))
+
+flags.DEFINE_enum('location',
+                  US_LAS,
+                  [US_LAS, DE_FKB, DE_FRA],
+                  ('Location of data center to be provisioned (us/las, '
+                   'de/fkb, de/fra)'))
+
+flags.DEFINE_integer('profitbricks_ram',
+                     None,
+                     ('Amount of RAM for the new server in multiples '
+                      'of 256 MB.'))
+
+flags.DEFINE_integer('profitbricks_cores',
+                     None,
+                     ('Number of cores for the new server.'))
+
+flags.DEFINE_string('profitbricks_disk_type',
```

Comment:
I think when I was reviewing this I didn't notice that this was just for the boot volume (or at least not until I got further in). Yeah the disk flags are only really intended for scratch/data disks and not boot volumes so feel free to keep this - I'd suggest adding something to the name (or at the very least flag description) to make it obvious that this is for the boot volume